module_1/src/com/epam/cycles/CyclesTaskC.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.cycles;
+
+public class CyclesTaskC {
+
+    private static final int AMOUNT_OF_NUMBERS = 100;
+
+    public static void main(String[] args) {
+        int sum = 0;
+        for (int i = 1; i <= AMOUNT_OF_NUMBERS; i++) {
```

Comment:
I don't want to be so boring, but summary of the first 100 numbers = 0 + 1 + ... + 99, so 100 isn't included
```suggestion
        for (int i = 1; i < AMOUNT_OF_NUMBERS; i++) {
```